src/main/sensors/gyro.h

```diff
@@ -206,6 +206,12 @@ typedef struct gyroConfig_s {
     uint8_t dyn_lpf_curve_expo; // set the curve for dynamic gyro lowpass filter
     uint8_t  simplified_gyro_filter;
     uint8_t  simplified_gyro_filter_multiplier;
+
+    uint8_t gyro_cal_manual;
+    uint8_t gyroCalibrated;
+    float gyroZeroX;
+    float gyroZeroY;
+    float gyroZeroZ;
```

Comment:
Floats are not the proper way to persist this data. Floats can't be exposed through the settings. This then adds the complication that a user's `diff` won't output the saved calibration and allow it to be persisted. See `acc_calibration` or `mag_calibration` for examples how this should be persisted and exposed in the settings.